CMakeLists.txt

```diff
@@ -131,7 +131,7 @@ add_subdirectory(src/Core)
 
 ## The source_group line creates the full visual studio filter layout
 get_target_property(sources Etterna SOURCES)
-source_group(TREE ${CMAKE_CURRENT_SOURCE_DIR} PREFIX "Etterna" FILES ${sources})
```

Comment:
It is very worth noting I have not seen how much this changed line affects Visual Studio

Reply:
Locally CMAKE_CURRENT_SOURCE_DIR and CMAKE_SOURCE_DIR are the same value so this change should do nothing to normal builds